keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -54,11 +58,13 @@ class DeliveryConfigController(
       description = "The delivery config. If its `name` matches an existing delivery config the operation is an update, otherwise a new delivery config is created."
     )
     deliveryConfig: SubmittedDeliveryConfig
-  ): DeliveryConfig {
-    validator.validate(deliveryConfig)
-    log.debug("Upserting delivery config '${deliveryConfig.name}' for app '${deliveryConfig.application}'")
-    return repository.upsertDeliveryConfig(deliveryConfig)
-  }
+  ): DeliveryConfig =
+    deliveryConfigProcessors.applyAll(deliveryConfig)
+      .let { processedDeliveryConfig ->
+        validator.validate(processedDeliveryConfig)
+        log.debug("Upserting delivery config '${processedDeliveryConfig.name}' for app '${processedDeliveryConfig.application}'")
+        repository.upsertDeliveryConfig(processedDeliveryConfig)
+      }
```

Comment:
Any `DeliveryConfigProcessor` instances are applied here in the controller. Note that `import` also delegates to this endpoint.